tests/foreman/cli/test_setting.py

```diff
@@ -33,315 +34,352 @@
 from robottelo.decorators import run_in_one_thread
 from robottelo.decorators import tier2
 from robottelo.decorators import tier3
-from robottelo.test import CLITestCase
 
 
-class SettingTestCase(CLITestCase):
-    """Implements tests for Settings for CLI"""
+@pytest.mark.stubbed
+@tier2
+def test_negative_update_hostname_with_empty_fact():
+    """Update the Hostname_facts settings without any string(empty values)
+
+    :id: daca0746-75ad-42fb-97ed-6db0224eeeac
+
+    :expectedresults: Error should be raised on setting empty value for hostname_facts setting
+
+    :CaseAutomation: notautomated
+    """
+
+
+@tier2
+@pytest.mark.parametrize('setting_update', ['discovery_prefix'], indirect=True)
+def test_positive_update_hostname_prefix_without_value(setting_update):
+    """Update the Hostname_prefix settings without any string(empty values)
+
+    :id: a84c28ea-6821-4c31-b4ab-8662c22c9135
+
+    :expectedresults: Hostname_prefix should be set without any text
+
+    :parametrized: yes
```

Comment:
@mshriver identified that `:parametrized: yes` should not be written before `:expectedresults:` something wrong happening in Polarian